ansible/roles/ros_install/tasks/main.yml

```diff
@@ -53,3 +53,6 @@
 
 - name: rosdep init
   command: rosdep init creates=/etc/ros/rosdep/sources.list.d/20-default.list
+  
+- name: Update dependencies
+  shell: bash -c "rosdep update -y"
```

Comment:
@ugocupcic in most cases which I have see in the code it was used shell: bash -c. So I decided to use the same approach. There was no specific reason just because it worked before.